### ISSUE TITLE
Connection to back end instance failed 

### ISSUE THREAD

**Issue by nareshm1** [healenium/healenium-web#62]
Hi @GannaChernyshova @moinuddin14 @valfirst @LittleDecorator @DzmitryHumianiuk @YauheniPo 

Connection to backend instance is failed  : 

Stack Trace : 
[INFO] --- hlm-report-mvn:1.1:initReport (hlmReport) @ mven.heal ---
[INFO] Healenium server Port = 7878
[INFO] Healenium server Host = localhost
Sep 14, 2020 5:46:47 AM com.epam.healenium.client.HealingClient makePost
WARNING: Failed to perform POST request. Reason: Failed to connect to localhost/0:0:0:0:0:0:0:1:7878
[INFO] sessionKey= null
[WARNING] Couldn't obtain session key from server!

**Heal.prop file:**
recovery-tries=3
score-cap=.6
heal-enabled=true
serverHost = 192.168.99.103
serverPort = 7878

![image](https://user-images.githubusercontent.com/67036539/93032261-3bd63f00-f64e-11ea-9795-c34a691dd0b6.png)
![image](https://user-images.githubusercontent.com/67036539/93032272-4e507880-f64e-11ea-96ad-e178b31335c9.png)
![image](https://user-images.githubusercontent.com/67036539/93032292-61fbdf00-f64e-11ea-80ea-4484ccc5f14f.png)

**Comment by Aliaksei-Ashukha** [comment]
Hi @nareshm1, 
Before to build the project you need to launch following services: healenium/hlm-backend and postgres (db)
Please, navigated to \src\test\resources and execute docker-compose.yaml

**Comment by nareshm1** [comment]
Hi @Aliaksei-Ashukha 

Yeah I have started them at first later on I executed the tests

Steps Followed:
1. In \src\test\resources  executed docker-compose up -d 
2. Later updated my maven
3. Right click on project --> Run as --> Maven test
4. Test is running and locators are also healed but the issue is it is unable to connect the server to **hlm report generation**
then I am getting the error.

![image](https://user-images.githubusercontent.com/67036539/93257792-056a0280-f7bb-11ea-8cd6-a54976a5e32f.png)

![image](https://user-images.githubusercontent.com/67036539/93257839-187cd280-f7bb-11ea-8539-65d26efe286f.png)

![image](https://user-images.githubusercontent.com/67036539/93257901-31858380-f7bb-11ea-9501-4e7d0330672c.png)
